satpy/readers/hsaf_nc.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019.
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Reader for H SAF blended precipitation products in NetCDF format.
+
+These products provide instantaneous precipitation rate estimates derived from
+a blending of geostationary (GEO) infrared and low-Earth-orbit (LEO) microwave
+observations using the H SAF "Rapid Update" algorithm.
+
+Currently, this reader supports the following products:
+  * **H60B** – Blended GEO/IR and LEO/MW instantaneous precipitation over the
+    full Meteosat (0°) disk.
+  * **H63** – Blended GEO/IR and LEO/MW instantaneous precipitation over the
+    Meteosat IODC (Indian Ocean) disk.
+  * **H90** – Accumulated Precipitation at ground by blended MW and IR IODC
+    over the Meteosat IODC (Indian Ocean) disk.
+
+Notes:
+    - Externally compressed files with the ``.gz`` suffix are automatically
+      uncompressed to the same directory and deleted on reader close.
+    - The reader relies on area definitions provided in the accompanying YAML
+      configuration file.
+    - Supports ``rain_rate`` and ``q_ind#`` datasets.
```

Comment:
```suggestion
    - Supports ``rain_rate`` and ``q_ind`` datasets.
```
I guess the hashtag is a typo?